### PR TITLE
BMP file loading fixes

### PR DESCRIPTION
**Pull request by SNMetamorph**

### CODE REVIEW

File: engine/common/imagelib/img_bmp.h [hunk 1]
```diff
@@ -25,7 +25,12 @@ GNU General Public License for more details.
 #define BI_FILE_HEADER_SIZE 14
 #define BI_SIZE 40 // size of bitmap info header.
 #if !defined(BI_RGB)
-#define BI_RGB 0 // uncompressed RGB bitmap(defined in wingdi.h)
+#define BI_RGB		  0	// uncompressed RGB bitmap (defined in wingdi.h)
```
Comment (nekonomicon) [inline engine/common/imagelib/img_bmp.h:28]: May be use enumeration?

File: engine/common/imagelib/img_bmp.c [hunk 2]
```diff
@@ -174,7 +180,7 @@ qboolean Image_LoadBMP( const char *name, const byte *buffer, fs_offset_t filesi
 		break;
 	}
 
-	estimatedSize = ( buf_p - buffer ) + ( image.width + padSize ) * image.height * ( bhdr.bitsPerPixel >> 3 );
+	estimatedSize = ( buf_p - buffer ) + image.width * image.height * ( bhdr.bitsPerPixel >> 3 );
```
Comment (a1batross) [inline engine/common/imagelib/img_bmp.c:183]: Why did you remove padding here?

Comment (SNMetamorph) [reply engine/common/imagelib/img_bmp.c:183]: It causes invalid estimated size calculation on images which dimensions is not power of 2. And therefore throws error.

File: engine/common/imagelib/img_bmp.h (hunk 1 above)
Comment (SNMetamorph) [reply engine/common/imagelib/img_bmp.h:28]: But why?
